packages/ag-charts-community/src/util/bezier.ts

```diff
@@ -0,0 +1,46 @@
+// Find the extreme points where the derivative is zero
+export function calculateDerivativeExtrema(p0: number, p1: number, p2: number, p3: number): number[] {
```

Comment:
Nice!

We have a similar bezier file at packages/ag-charts-community/src/scene/util/bezier.ts . Not too bothered where the file ends up, but would be good to merge them

Reply:
Done.